wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController.swift

```diff
@@ -23,16 +23,35 @@ import WireSyncEngine
 
 final class ConversationListViewController: UIViewController {
 
+    // MARK: - Properties
+
     let viewModel: ViewModel
 
-    /// private
     private var viewDidAppearCalled = false
     private static let contentControllerBottomInset: CGFloat = 16
 
+    private var filterContainerView: UIView!
+    private var filterLabel: UILabel!
+    private var removeButton: UIButton!
```

Comment:
Can we avoid implicitly unwrapped types by initializing them (also lazily?) and setting up later when the view loads?

Reply:
Done: https://github.com/wireapp/wire-ios/commit/6a9b3d3c3cdae9243a9e600dc5a8ce7508ba52c6